test/integration/targets/postgresql/tasks/main.yml

```diff
@@ -268,92 +268,23 @@
     that:
       - "result.changed == False"
 
-- name: Create a user with all role attributes
+# BYPASSRLS role attribute was introduced in Postgres 9.5, so 
+# we want to test atrribute management differently depending
+# on the version. See https://github.com/ansible/ansible/pull/24625
+# for more details.
+- name: Get Postgres version 
```

Comment:
You could add a debug task here in order to print the PostgreSQL version tested. The version appears only when verbose is enabled.